contracts/WamplLoanRouter.sol

```diff
@@ -0,0 +1,111 @@
+pragma solidity ^0.8.3;
+
+import "./interfaces/ILoanRouter.sol";
+import "./interfaces/IBondController.sol";
+import "./interfaces/ITranche.sol";
+import "./interfaces/IButtonWrapper.sol";
+import "./interfaces/IWAMPL.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
```

Comment:
nit: separate external imports from local ones

Reply:
Can be a separate PR but solmate SafeTransferLib is _significantly_ more gas efficient https://github.com/Rari-Capital/solmate/blob/main/src/utils/SafeTransferLib.sol